servicetemplates/https%3A%2F%2Fust-quantil.github.io%2Fquantum%2Fapplications%2Fservicetemplates/Shor-UI_Docker-w1/ServiceTemplate.tosca

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Definitions xmlns="http://docs.oasis-open.org/tosca/ns/2011/12" xmlns:winery="http://www.opentosca.org/winery/extensions/tosca/2013/02/12" xmlns:selfservice="http://www.eclipse.org/winery/model/selfservice" xmlns:testwineryopentoscaorg="http://test.winery.opentosca.org" targetNamespace="https://ust-quantil.github.io/quantum/applications/servicetemplates" id="quantumApplicationsServiceTemplates-Shor-UI_Docker-w1">
+    <ServiceTemplate name="Shor-UI_Docker-w1" targetNamespace="https://ust-quantil.github.io/quantum/applications/servicetemplates" id="Shor-UI_Docker-w1">
+        <TopologyTemplate>
+            <NodeTemplate xmlns:otntyIgeneral="http://opentosca.org/nodetypes" name="DockerEngine_w1" minInstances="1" maxInstances="1" type="otntyIgeneral:DockerEngine_w1" id="DockerEngine_w1_0" winery:x="486" winery:y="630">
```

Comment:
Shouldn´t we use the prefixes defined here? 
https://github.com/OpenTOSCA/tosca-definitions-common/blob/main/admin/namespaces/Namespaces.json